examples/AzureSDKDemoSwiftUI/AzureSDKDemoSwiftUI/Pages/BlobDownloadView.swift

```diff
@@ -0,0 +1,140 @@
+// --------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// The MIT License (MIT)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the ""Software""), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+//
+// --------------------------------------------------------------------------
+
+import SwiftUI
+import UIKit
+
+import AzureCore
+import AzureIdentity
+import AzureStorageBlob
+
+import AVKit
+import Photos
+
+final class BlobDownloadTableViewController: UIViewControllerRepresentable {
+    var viewController: UITableViewController?
```

Comment:
So what is the reason for keeping these UIViewControllerRepresentable instead of a typical SwiftUI View struct?

Reply:
If it needs to be this, that's fine. We can consider a baseline for improvement. But if it isn't needed, this should look as close to "pure" SwiftUI as possible.

Reply:
Yes we can discuss this but as I started working that I noticed that the video player further down would require a `UIViewRepresentable` and I would eventually go down this path. IMO this is the cleaner implementation